@navikt/core/react/src/util/composeEventHandlers.ts

```diff
@@ -0,0 +1,22 @@
+/* https://github.com/radix-ui/primitives/blob/main/packages/core/primitive/src/primitive.tsx#L1 */
+
+/**
+ * Utility to consistently call original eventhandler, often from props and internal eventhandler
+ * @internal
+ */
+export function composeEventHandlers<T>(
+  originalEventHandler?: (event: T) => void,
+  ourEventHandler?: (event: T) => void,
+  { checkForDefaultPrevented = true } = {},
+) {
+  return function handleEvent(event: T) {
+    originalEventHandler?.(event);
+
+    if (
+      checkForDefaultPrevented === false ||
+      !(event as unknown as Event).defaultPrevented
+    ) {
+      return ourEventHandler?.(event);
```

Comment:
Det at vi nå lytter til `defaultPrevented` flere steder kan vel i teorien brekke ting...? Men det å gjøre `preventDefault()` har vel ikke hatt noen effekt tidligere, så det er vel usannsynlig at folk har gjort det? (Ble mye "vel" her 😅)

Reply:
Ja endringen har potensiale for å kunne brekke noe, selv om det vil være edgecaser 🤔 Er med på å stashe denne på en v6-branch hvis vi ikke ønsker å risikere det som en minor-versjon